features/step_definitions/file_steps.coffee

```diff
@@ -5,17 +5,21 @@ path = require 'path'
 module.exports = ->
 
   @Given /^I have no test files$/, (done) ->
-    done() # No op for readability
+    done()
 
 
-  @Given /^I have 1 (passing|failing) test$/, (status, done) ->
-    outputTestFile path.join(@tmpDir, 'example_spec.coffee'), passing: status is 'passing', done
+  @Given /^I have a (passing|failing) test$/, (status, done) ->
+    outputTestFile path.join(@tmpDir, 'example_spec.coffee'),
+                   passing: status is 'passing',
+                   done
```

Comment:
1. I disagree, especially with the abundant difference. Yeah its similar to a code block but I don't think that plays a part. I see minimal difference in clarity between the following:
  ```coffee
  email.send(
    new Smtp().instance(),
    new Proxy().connect('gateway'))

  # vs

  email.send new Smtp().instance(),
             new Proxy().connect('gateway')
  ```

2. Yes, aligning to the left requires putting the first argument onto the next line which requires parenthesis to make it valid coffeescript. This is inconsistent though as if the first argument is an object, the parens are not needed.
  ```coffee
  # valid
  foo
    bar: baz

  # invalid
  foo
    bar
  ```

3. same as 2

4. Yes, aligning to the left requires putting the first argument onto the next line. Not a big deal to me. 

5. Yes it does. To fully utilize the horizontal space, just put all the arguments on the same line. Presumably you don't want to do that (which is why we are having this discussion). Not important to me as it makes the method call stand out more.

6. Yes that an interesting difference. Having the singular case be special isn't an uncommon thing though. Chaining methods produces a similar thing. Also objects when parens are required (ruby)
  ```
  foo.bar()

  foo
    .bar()
    .baz()

  {foo: bar}
  
  {
    foo: bar
    bizz: buzz
  }
  ```

7. I disagree, anything indented belongs to the first line above with less indentation. And the parenthesis required by coffeescript lets you know its a function call and the arguments are still to come.

---

In school I had a professor that said right aligning code was "wasting time". Things will change and you need to realign. Sure this is easier with modern text editors but overall its harder to maintain. The example my professor was more targeted at:
```coffee
foo:         bar1
longLongFoo: bar2
```
Where if the longest name changes, you need to realign the world. But I apply it to this as well due to renames of the function call
```coffee
email.send new Smtp().instance(),
           new Proxy().connect('gateway'))

# rename send to sendWithProxy with a find and replace in my project

email.sendWithProxy new Smtp().instance(),
           new Proxy().connect('gateway'))
```
Things are now misaligned. This happened all the time on a project and those differences need to fixed by hand.